src/main/scala/com/actian/spark_vector/datastream/writer/DataStreamWriter.scala

```diff
@@ -68,6 +68,9 @@ object DataStreamWriter extends Logging {
   /** Write an ASCII encoded string to `out` */
   def writeString(out: DataOutputStream, s: String): Unit = writeByteArray(out, s.getBytes("ASCII"))
 
+  /** Write a null-terminated ASCII encoded string to `out`, without its preceding length */
+  def writeStringV2(out: DataOutputStream, s: String): Unit = { out.write(s.getBytes("ASCII")); out.write(0) }
```

Comment:
`writeStringWithNullterm` ?

also you could indent it in two lines (imo no need for the `;` to make a 1-liner in this case)